doc/examples/applications/plot_image_comparison.py

```diff
@@ -33,7 +33,7 @@
 # The `checkerboard` method alternates tiles from the first and the second
 # images.
 
-fig = plt.figure(figsize=(8, 9))
+fig, ax = plt.subplots(figsize=(8, 9))
```

Comment:
I think using `fig, ax = plt.subplots` over `fig = plt.figure` is actually more confusing here since `ax` is never used below. `plt.figure` might be fine as long as the returned object -- the figure -- is captured and used...

Reply:
I see your point. I must say that this example was quite peculiar due to the use of the GridSpec and I was not really sure how to handle it. It is also the example where I left plt.plot(). But, I guess, also in this example plt.plot can be removed and plt.show() added at the end of the file, as for the other examples in the folder.

Reply:
Reverted